Content.Server/PDA/Ringer/RingerComponent.cs

```diff
@@ -22,7 +22,7 @@ public sealed class RingerComponent : Component
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("range")]
-        public float Range = 0.5f;
+        public float Range = 3f;
```

Comment:
You sure the increase wont lead to a lot of noise pollution?

Reply:
The previous PR reduced this from 3 to .5 and it got merged before the OG author could change it back.